src/domain/communication/communication/communication.resolver.fields.ts

```diff
@@ -19,9 +20,28 @@ export class CommunicationResolverFields {
   })
   @Profiling.api
   async discussions(
-    @Parent() communication: ICommunication
+    @Parent() communication: ICommunication,
+    @Args({
+      name: 'limit',
+      type: () => Float,
+      description:
+        'The number of Discussions to return; if omitted return all Discussions.',
+      nullable: true,
+    })
+    limit: number,
+    @Args({
+      name: 'orderBy',
+      type: () => DiscussionsOrderBy,
+      description: 'The sort order of the Discussions to return.',
+      nullable: true,
+    })
+    orderBy: DiscussionsOrderBy
```

Comment:
Can you move this to a single optional input object with optional params - it's easier to mantain and use